cms/core/forms.py

```diff
@@ -98,3 +100,33 @@ def clean_corrections(self) -> StreamValue:  # noqa: C901
 
 class NoLocaleFilterInChoosersForm(SearchFilterMixin, BaseFilterForm):
     """A chooser filter form that deliberately excludes the locale filter."""
+
+
+class PageWithEquationsAdminForm(WagtailAdminPageForm):
+    def _process_content_block(self, block: StreamValue) -> None:
+        if block.block_type == "equation":
+            equation = block.value["equation"]
+            # Remove $$ from the start and end of the equation
+            if equation.startswith(FORMULA_INDICATOR) and equation.endswith(FORMULA_INDICATOR):
+                equation = equation[2:-2]
+            try:
+                block.value["svg"] = latex_formula_to_svg(equation)
+            except RuntimeError:
```

Comment:
we should log the original exception to the system log

Reply:
Logging added in [ff3d727](https://github.com/ONSdigital/dis-wagtail/pull/192/commits/ff3d72796fbed52aade83a81b7ff427c7ebd643f)